### PR TITLE
STUD-190: Update support nav copy

### PR DESCRIPTION
**Pull request by escobarjonatan**

### CODE REVIEW

File: apps/studio/src/pages/home/owners/OwnersTable.tsx [hunk 1]
```diff
@@ -212,7 +212,7 @@ export default function OwnersTable({
                       `${firstName} ${lastName}`
                     ) : (
                       <Typography fontStyle="italic" fontWeight={ 400 }>
-                        Waiting on account creation
+                        Waiting on collaborator&apos;s acceptance
```
Comment (scandycuz) [inline apps/studio/src/pages/home/owners/OwnersTable.tsx:215]: 👍

**Review by scandycuz — APPROVED** [review]

File: apps/studio/src/pages/home/owners/OwnersTable.tsx [hunk 2]
```diff
@@ -212,7 +212,7 @@ export default function OwnersTable({
                       `${firstName} ${lastName}`
                     ) : (
                       <Typography fontStyle="italic" fontWeight={ 400 }>
```
Comment (dmkirshon) [inline apps/studio/src/pages/home/owners/OwnersTable.tsx:214]: This italics is a bit strange and seems out of place to other font styles we've used... can't find any reference to it on Figma either.

**Review by dmkirshon — APPROVED** [review]
LGTM, just comment about UI font style

File: apps/studio/src/pages/home/owners/OwnersTable.tsx (hunk 2 above)
Comment (escobarjonatan) [reply apps/studio/src/pages/home/owners/OwnersTable.tsx:214]: Looks like the mocks are also different and dont include either scenario. We can do that as a separate update when we add icons as part of collaborator status

**Pull request merged**